data-raw/Massachusetts 2020 vs. 2020 Traffic Data Comparison - Traffic Data 2020 vs 2019.csv

```diff
@@ -0,0 +1,265 @@
+Community,Functional Class,Change_in_Speed,Change_in_Volume,Change_in_Accidents

```

Comment:
This file shouldn't be in the commit. In the import script, you are reading from the other CSV. You shouldn't have both of them included.